File: CounterStrikeDashboard.Core/Model/PlayerTuple.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CounterStrikeDashboard.Core.Model
{
    public class PlayerTuple
    {
        public string Name { get; set; }
        public string UniqueIdentifier { get; set; }

        public int Kills { get; set; }
        public int Deaths { get; set; }

        public bool Connected { get; set; }
    }
}

File: CounterStrikeDashboard/Client/App/Models/PlayerTuple.ts
module csdash {
    export class PlayerTuple {
        private __name__ = "PlayerTuple";

        name: string = "Unknown";
        uniqueIdentifier: string = "";
        team: string = "none";
        kills: number = 0;
        deaths: number = 0;
        connected: boolean = false;
    }
}